src/themes/default/elements/icon.variables

```diff
@@ -83,1463 +83,1463 @@ Icons are order A-Z in their group, Solid, Outline, Thin (Pro only) and Brand
 /* Deprecated *In/Out Naming Conflict) */
 @icon-deprecated-map: {
     linkedin_in:\f0e1;
-    zoom_in: \f00e;
-    zoom_out: \f010;
-    sign_in: \f2f6;
-    in_cart: \f218;
-    log_out: \f2f5;
-    sign_out: \f2f5;
+    zoom_in: '\f00e';
```

Comment:
fixes https://github.com/fomantic/Fomantic-UI/pull/2594#discussion_r1048969446

the mentioned export/generator lib should be fixed to export the data with single quotes and no space before colon and one space after

Reply:
Done by https://github.com/fomantic/create-fomantic-icons/pull/215